test/test-mda-models/test-data-delivery-spark-model-basic/pom.xml

```diff
@@ -136,7 +136,7 @@
             <classifier>tests</classifier>
             <type>test-jar</type>
             <scope>test</scope>
-            <version>${version.aissemble}</version>
+            <version>${project.version}</version>
```

Comment:
I: `release:prepare` was complaining that `version.aissemble` was unresolvable and so it didn't know how to update the version for this depenency.